js/tests/unit/dropdown.spec.js

```diff
@@ -54,6 +54,42 @@ describe('Dropdown', () => {
       expect(dropdown.toggle).toHaveBeenCalled()
     })
 
+    it('should create offset modifier correctly when offset option is a function', () => {
+      fixtureEl.innerHTML = [
+        '<div class="dropdown">',
+        '  <button class="btn dropdown-toggle" data-bs-toggle="dropdown">Dropdown</button>',
+        '  <div class="dropdown-menu">',
+        '    <a class="dropdown-item" href="#">Secondary link</a>',
+        '  </div>',
+        '</div>'
+      ].join('')
+
+      const getOffset = () => [10, 20]
```

Comment:
It is better if you create a spy and  then expect that to have been called after showing the dropdown :slightly_smiling_face:

Reply:
yeah, I tried but I'm not confident with javascript mocking yet.
I'm working on it

Reply:
The funny thing is that the `shown.bs.dropdown` is not related to Popper, so in order to test that a getOffset spy function has been called showing the dropdown I've to use the `popperConfig.onFirstUpdate` attribute.